python/paddle/fluid/tests/unittests/collective/CMakeLists.txt

```diff
@@ -0,0 +1,230 @@
+set(LOCAL_ALL_ARCH ON)
```

Comment:
Add lines
# licenses
# WARNING comment

Reply:
1. no need of licenses